src/main/java/seedu/address/logic/parser/GroupCommandParser.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.logging.Logger;
+
+import seedu.address.MainApp;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.AddToGroupCommand;
+import seedu.address.logic.commands.GroupCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.group.Group;
+
+/**
+ * Parses input arguments and creates a new GroupCommand object
+ */
+public class GroupCommandParser implements Parser<GroupCommand> {
+
+    private static final Logger logger = LogsCenter.getLogger(MainApp.class);
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the GroupCommand
+     * and returns an GroupCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public GroupCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        Group groupToDisplay;
+
+        String[] splitArgs = args.trim().split("\\s+", 2);
+        if (splitArgs.length != 1) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddToGroupCommand.MESSAGE_USAGE));
```

Comment:
groupcommand is throwing an addtogroupcommand message usage rather than its own groupcommand messsage usage which can be confusing